docs/platform_management_plan/project_management.rst

```diff
@@ -0,0 +1,165 @@
+
+  * Review and approval of *Contribution Requests*
+  * Project management, e.g., creation of the roadmap
+  * High-level project control and coordination between multiple software modules.
+  * Escalation instance for software module project leads and committers.
+
+  *Technical lead circle* proposes and elects a *Technical lead circle Assistant* and his deputy with bare majority, who is responsible for scheduling and announcing meetings, preparing and announcing agenda, writing meeting minutes and protocols. The *Technical lead circle Assistant* and his deputy can resign anytime on their own will.
+
+Technical committees
+--------------------
+* **Communities**
+
+  *Communities* allow committers and contributors to exchange their
+  opinions, take architectural decisions and implement the topics of some special
+  technical domain, e.g. testing tooling. One of the *Communities*' important tasks
+  is to do define architecture, component requirements (if applicable)
+  and do a breakdown to the concrete tasks of the platform sagas (see `Planning`_) .
+  Currently following *Communities* are defined in the *SCORE* project:
+
+  * *Tooling/Infrastructure*: *community* for all kind of infra topics:
+    compiler, IDE, build toolchain and e.t.c.
+  * *Testing*: *community* to clarify questions and define testing strategy
+    for the 'SCORE' project
+  * *Software Architecture*: *community* for clarification of software architecture topics,
+    e.g. discussion of new features or coding guidelines.
+  * *Software Development Process*: *community* for definition and maintaining
+    of safety, security and quality software development process.
+
+  The planning of the activities is done by every *Community* independent of other
+  teams. Each *Community* has a *Community Lead*, who is nominated by the *Technical lead circle*.The prioritization of some topics can be requested by the *Technical lead circle*
+  in order to achieve milestones on time. All important architectural decisions
+  should be reported to the project as *Contribution Request*
+  to get the final approvement from the *Technical lead circle*.
+
+* **Cross functional teams**
+
+  *Cross functional teams* are responsible for some piece
+  of work from the beginning (e.g. definition of the architecture) till the end
+  (e.g. integration tests) and are assigned to one particular software module. *Cross-functional teams* work independently from each other on *GitHub* issues in the assigned software module. *Cross-functional teams* consist of the contributors, who can specify requirements, define architecture, develop source code and implement tests afterwards. *Project Leads* and *Committers* are also *Contributors* and effectively work on processing of *GitHub* issues.
```

Comment:
We also need one "cross functional team" working on the SCORE SW platform level (as there we also need the Safety, Quality ... roles